hack/pressure-test/README.md

```diff
@@ -0,0 +1,16 @@
+# Pressure Test Guide
+
+## Files
+
+Two application templates are provided in `app-templates` folder. The community template is a standard template which can be applied in standard KubeVela environment. The sae template, on the other hand, requires support for extended definitions (env & cloneset-service) and OpenKruise to be installed. The `tolerate-hollow-node` trait is used to add toleration to pods in pressure test applications, which makes them allocatable for hollow nodes mocked by `kubemark`.
+
+The `dashboard.json` is a grafana dashboard file that can track several groups of key metrics when running pressure tests.
+
+## Steps
+
+1. **Prepare Kubernetes Clusters.** The pressure test is conducted under ACK (Alibaba Cloud Container Service for Kubernetes) environment. You can set up your own Kubernetes cluster environment if you would like to run your own pressure test. 
+2. **Prepare Kubernetes Environments.** You need to install KubeVela first in your environment and add environment variable `PERF=1` to the pods of KubeVela to enable performance monitoring. Remember to disable MutatingWebhookConfigurations and ValidatingWebhookConfigurations if you only care about the KubeVela controller performance.
+3. **[Optional] Prepare Kubemark Environments.** The `tolerate-hollow-node` trait need to be installed if you would like to make pressure test on hollow nodes mocked by `kubemark`. Another issue for using `kubemark` is that you need to ensure pods created by pressure test applications would be assigned to hollow nodes while other pods like KubeVela controller must not be assigned on these hollow nodes. You can achieve that by adding taints to nodes and adding nodeSelector or toleration to pods.  
+4. **Prepare Monitoring Tools.** Install Grafana, Loki, Prometheus in your Kubernetes which can help you watch pressure test progress and discover problems.
```

Comment:
Use `observability` addon?

Reply:
> Use `observability` addon?

Ok, I added it. Using addon will require user to install `vela` first. The usage of it is introduced as an optional choice.